packages/sanity/src/core/store/_legacy/project/projectStore.ts

```
@@ -26,6 +26,9 @@ const getOrganizationId = memoize(
         map((res) => res.organizationId),
         repeat({delay: REFETCH_INTERVAL}),
         shareReplay(1),
+        catchError(() => {
+          return of(null)
```

This will swallow any type of error (not just `401`), but seems reasonable for now.

---

Yes, that was the idea. The studio should not crash when opening a document, regardless the error received here. 
We can do a follow up and be more specific on how the ui displays this if we need it.